### PR TITLE
macos: add Window>Show UTM menu option

### PR DESCRIPTION
**Pull request by garamb1**
As mentioned in issue https://github.com/utmapp/UTM/issues/5898, I also noticed there is no way of getting only the virtual machine list window to open if:
 - UTM is running one of more virtual machines
 - The virtual machine list is closed
 - The menu bar icon & menu are disabled (hence the user cannot use CMD + 0 or click on `Show UTM` from the menu)

This PR adds the option in the Window Menu to show UTM, also adding the CMD + Alt + L keyboard shortcut to do so.

The main window is shown by sending a new notification, handled in the macOS `homeWindow` View. I am not sure whether this would be the best way of doing so with SwiftUI, so any suggestions are welcome! Thanks for the amazing app!

### CODE REVIEW

**Comment by garamb1** [comment]
@osy thanks for adding this to the next release!
I had to cherry pick my original commit and merge the changes from main, it should be ok now.

File: Platform/macOS/UTMApp.swift [hunk 1]
```diff
@@ -32,6 +32,9 @@ struct UTMApp: App {
                     data.showErrorAlert(message: message)
                 }
             }
+            .onReceive(NSNotification.ShowMainWindow) { _ in
+                NSApplication.shared.windows.first?.makeKeyAndOrderFront(nil)
```
Comment (osy) [inline Platform/macOS/UTMApp.swift:36]: I don't know if this is good enough. I might play around with it but thinking of capturing a reference in the AppDelegate in order to reference it.

Comment (garamb1) [reply Platform/macOS/UTMApp.swift:36]: Sounds good, thanks!

**Comment by osy** [comment]
I just realized there is already an option to show the main window:
image

Also, I don't understand this:
> The menu bar icon & menu are disabled

image

When you disable the dock icon, the menu icon is forced to be enabled.

**Comment by garamb1** [comment]
I think that was the case on previous releases before the introduction of UTM Server in the Windows menu, see screenshot for reference:
image

As the issue is gone now, closing the PR.
Thank you @osy

**Comment by osy** [comment]
Ah I see, that's a weird coincidence lol.

**Pull request closed without merging**